teacher/templates/session_view_teacher.html

```diff
@@ -0,0 +1,94 @@
+{% extends "teacher_base.html" %}
+{% load crispy_forms_tags %}
+
+{% block content %}
+
+<div style="text-align: center;">
+  <div style="width: 85%; display: inline-block" class="mui-panel">
+    <h1>{{ classroom.course }} - {{ session.date }}</h1><hr>
+      {% if session %}
+        <h1>{{ session.title }}</h1>
+        <form action="{% url 'take_attendance' %}" method="GET">
```

Comment:
This could be a link instead of a form

Reply:
Resolved 495857a4bdcf59646cb8478b975856430ee980c7